content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md

```diff
@@ -7,46 +7,106 @@ content_template: templates/task
 
 {{% capture overview %}}
 
-This page explains how to manage certificates manually with kubeadm.
+Client certificates generated by [kubeadm](/docs/reference/setup-tools/kubeadm/kubeadm/) expire after 1 year. This page explains how to manage certificate renewals with kubeadm.
 
 {{% /capture %}}
 
 {{% capture prerequisites %}}
 
-These are advanced topics for users who need to integrate their organization's certificate infrastructure into a kubeadm-built cluster. If kubeadm with the default configuration satisfies your needs, you should let kubeadm manage certificates instead.
+Be familiar with [PKI certificates and requirements in Kubernetes](/docs/setup/certificates/).
 
-You should be familiar with [PKI certificates and requirements in Kubernetes](/docs/setup/certificates/).
+Have a working Kubernetes cluster installed using kubeadm with your certificates stored in `/etc/kubernetes/pki` folder; in case you are using a different location, the following can be adapted accordingly.
 
 {{% /capture %}}
 
 {{% capture steps %}}
 
-## Renew certificates with the certificates API
+## Check certificate expiration
 
-The Kubernetes certificates normally reach their expiration date after one year. 
+`check-expiration` can be used to check certificate expiration.
 
-Kubeadm can renew certificates with the `kubeadm alpha certs renew` commands; you should run these commands on control-plane nodes only.
+```
+kubeadm alpha certs check-expiration
+```
+
+The output is similar to this:
+
+```
+CERTIFICATE                EXPIRES                  RESIDUAL TIME   EXTERNALLY MANAGED
+admin.conf                 May 15, 2020 13:03 UTC   364d            false
+apiserver                  May 15, 2020 13:00 UTC   364d            false
+apiserver-etcd-client      May 15, 2020 13:00 UTC   364d            false
+apiserver-kubelet-client   May 15, 2020 13:00 UTC   364d            false
+controller-manager.conf    May 15, 2020 13:03 UTC   364d            false
+etcd-healthcheck-client    May 15, 2020 13:00 UTC   364d            false
+etcd-peer                  May 15, 2020 13:00 UTC   364d            false
+etcd-server                May 15, 2020 13:00 UTC   364d            false
+front-proxy-client         May 15, 2020 13:00 UTC   364d            false
+scheduler.conf             May 15, 2020 13:03 UTC   364d            false
+```
+
```

Comment:
We should include  a note snippet or block that states: "Kubelet.conf is managed by kubelet itself (kubeadm configures kubelet for automatic certificate renewal)" or similar

Reply:
Done, at the end of the paragraph